graphql-mode.el

```diff
@@ -343,7 +343,7 @@ This is the function to be used for the hook `completion-at-point-functions'."
        ;; the search can continue.
        ((graphql--in-arguments-p)
         (let* ((end (save-excursion (up-list) (point)))
-               (match (search-forward-regexp "\\(\\_<.+?\\_>\\):" end t)))
+               (match (search-forward-regexp "\\(\\_<[a-zA-Z0-9_-]+?\\_>\\):" end t)))
```

Comment:
I couldn't find online what exactly constitutes a valid gql parameter name, so I'm just guessing alphanumeric + `-_`.

Reply:
I was looking at the same thing!

I think here it is https://spec.graphql.org/June2018/#sec-Names

`/[_A-Za-z][_0-9A-Za-z]*/`

So close guess.

Should we try to use the exact one?

Reply:
Nice thanks, the `/` had me scratching my head for a bit until I found out they were javascript specific :smile: